chart/modeldb/charts/webapp/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ .Release.Name }}
+    component: webapp
+{{- if .Values.labels}}
+{{toYaml .Values.labels | indent 4 -}}
+{{- end}}
+  name: {{ .Release.Name }}-webapp
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+      component: webapp
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+        component: webapp
+{{- if .Values.labels}}
+{{toYaml .Values.labels | indent 8 -}}
+{{- end}}
+      annotations:
+        verta.ai/disable_anti_affinity: "true"
+    spec:
+      containers:
+        - image: {{ .Values.image }}:{{ .Values.imageTag }}
+          imagePullPolicy: Always
+          name: {{ .Release.Name }}
+          ports:
+            - containerPort: 3000
```

Comment:
we had these settings for AMI : https://github.com/VertaAI/modeldb/blob/master/chart/modeldb/templates/modeldb-frontend-service.yaml#L9-L12 do you think they will no longer be needed ?

Reply:
AMI is different, since the system is self contained, so opening a node port is less of an issue than doing that in your cluster (e.g. with that config by default, you couldn't have two separate sets of the webapp running).